modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/graphql/servlet/GraphQLServletExtender.java

```diff
@@ -1675,7 +1675,9 @@ private void _registerGraphQLDTOContributor(
 				FieldCoordinates.coordinates("mutation", createName),
 				(DataFetcher<Object>)
 					dataFetchingEnvironment -> graphQLDTOContributor.createDTO(
-						dataFetchingEnvironment.getArgument(resourceName),
+						_objectMapper.convertValue(
```

Comment:
Mmm... the getArgument method is tricky because it support several types... have you tested it with different? just debugging, to see if you can get a strange ClassCast.